cmd/openqa-revtui/openqa-revtui.go

```diff
@@ -598,6 +599,17 @@ func tui_main(tui *TUI, instance gopenqa.Instance) error {
 			// Shift through the sorting mechanism
 			tui.SetSorting((tui.Sorting() + 1) % 2)
 			tui.Update()
+		} else if key == 'o' {
+			for _, job := range tui.Model.jobs {
+				if !tui.hideJob(job) {
+					var err error
+					err = exec.Command("xdg-open", job.Link).Start()
```

Comment:
```suggestion
					err := exec.Command("xdg-open", job.Link).Start()
```

Let's stay go idomatic :slightly_smiling_face: (And please remove the previous line)

Reply:
done